content/en/docs/started/workstation/minikube-linux.md

```diff
@@ -153,8 +160,10 @@ cd ${KF_DIR}
 kfctl apply -V -f ${CONFIG_URI}
 ```
 
-When the installation finishes, run the following command to see whether all the pods are in running status. Depending on your machine’s capability, it may take a few minutes to reach full running status:
-```
+When the installation finishes, run the following command to see whether all the pods are in running status.
+Depending on your machine’s capability, it may take a few minutes to reach full running status:
+
+```SHELL
 kubectl get pod -n kubeflow
 ```
 
```

Comment:
Moving from the commit (line 164) to below the command line block:

```diff
...
kubectl get pod -n kubeflow
[end of block]

It may take a few minutes to reach full running status.
```